src/components/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -1,42 +1,66 @@
 import * as React from 'react';
-import {ToggleSwitch as CoreToggleSwitch, ToggleSwitchProps as CoreToggleSwitchProps} from 'wix-ui-core/ToggleSwitch';
-import {ThemedComponent} from 'wix-ui-theme';
-import {oneOf, Requireable} from 'prop-types';
-import {ThemedComponentProps} from 'wix-ui-jss';
-import {theme, Size, Skin} from './theme';
-import {WixComponentProps} from 'wix-ui-core/dist/src/createHOC';
-
-export interface ToggleSwitchProps extends CoreToggleSwitchProps {
-  /** size of the toggle switch */
-  size?: Size;
+import * as PropTypes from 'prop-types';
+import {
+  ToggleSwitch as CoreToggleSwitch,
+  ToggleSwitchProps as CoreToggleSwitchProps
+} from 'wix-ui-core/StylableToggleSwitch';
+import style from './ToggleSwitch.st.css';
+import {Skin, Size, SKINS, SIZES} from './constants';
+import {ToggleOff, ToggleOn, SmallToggleOff, SmallToggleOn} from 'wix-ui-icons-common/system';
+import {withStylable} from 'wix-ui-core/withStylable';
+import {omit} from 'lodash';
```

Comment:
some say we should `import omit from 'lodash/omit';`

Reply:
yeah, my bad